utils/common/service_liveness_check.yml

```diff
@@ -0,0 +1,8 @@
+#  This will be execute on both pre/post checks
+- name: Checking whether application pods are in running state
+  shell: kubectl get pod {{ liveness_checking_pod }} -n {{ namespace }} -o custom-columns=:.status.phase --no-headers
+  register: result
```

Comment:
Try to take different variables for different register as it is overriding the previous one.

Reply:
code has been changed